src/button/AnchorButton.jsx

```diff
@@ -2,25 +2,10 @@ import React from "react";
 import PropTypes from "prop-types";
 import ButtonStyles from "./button.style";
 
-export default function AnchorButton({
-  onClick,
-  type,
-  href,
-  target,
-  rel,
-  children,
-  customStyle
-}) {
+export default function AnchorButton({ children, type, ...rest }) {
   return (
     <ButtonStyles>
-      <a
-        className={type}
-        onClick={onClick}
-        href={href}
-        target={target}
-        rel={rel}
-        style={customStyle}
-      >
+      <a {...rest} className={type}>
```

Comment:
what is rest?

Reply:
rest is the rest of the props that might get parsed. 
This includes everything. `style={{color:'red'}}`, `innerRef={()=>{}}`, `rel='nonopener'` etc etc

Reply:
Can't we just define the rel='noopener' inside the a tag itself. I get that the ...rest passes in all props but I don't think its a good pattern to use. If we need to pass a lot of props liks this then this class should just be called custom button or something with a <Button>{children}</Button> feature. That way we're not extending it into the future to cover the many different ways to do a tags. Then people can just do <CustomButton><a href="whatever they want" /></CustomButton>. All people will really care about are the styles so it looks like a button.